src/app/pages/assistant/assistant.component.scss

```
@@ -0,0 +1,365 @@
+    height: 6px;
+    border-radius: 50%;
+    background: var(--text-secondary, #999);
+    animation: typingBounce 1.4s infinite ease-in-out;
+
+    &:nth-child(1) { animation-delay: 0s; }
+    &:nth-child(2) { animation-delay: 0.2s; }
+    &:nth-child(3) { animation-delay: 0.4s; }
+  }
+}
+
+@keyframes typingBounce {
```

Stylelint (17.3.0)

[error] Expected keyframe name "typingBounce" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)